examples/example_plugin/example_plugin/ui/index.js

```diff
@@ -1,11 +1,19 @@
 export { default as ExampleAppFullWidthPageComponent } from "./FullWidthPage"
+export { default as ExampleAppCustomRouteView } from "./CustomRouteView"
 export { default as ExampleAppOverrideModelView } from "./ModelView"
 
 const app_config = {
     detail_tabs: {},
     full_width_components: {
         "dcim:devices": ["ExampleAppFullWidthPageComponent"]
     },
+    /**
+     * The key of `routes_view_components` is the name of the url path,
+     * and the value is the View Component that should be associated with that url path.
+     */
+    routes_view_components: {
+        "home": "ExampleAppCustomRouteView",
```

Comment:
For clarification: This ultimately becomes `example_plugin:home`, correct?

Reply:
This route: https://github.com/nautobot/nautobot/blob/develop/examples/example_plugin/example_plugin/urls.py#L19 ?

Reply:
OK, let's clear up some conventions here. We include their own app label for overridden views but not for this. For now let's include it here, even if it's redundant.